Gemfile

```diff
@@ -1,11 +1,13 @@
 source 'https://rubygems.org'
 
 gemspec name: 'concurrent-ruby'
+gemspec name: 'concurrent-ruby-edge'
```

Comment:
When I was created the extension gem I ran into some problem with this. I can't remember what it was. That's why the extension gem isn't listed here. We'll need to take a look at this after we merge. Whatever it was may no longer be an issue.